### PR TITLE
Set up Travis CI

### PR DESCRIPTION
**Pull request by storresy**
**Summary**
- Added travis configuration file to run rubocop on every push
- Added travis build status badge to Readme 

**Trello card**
https://trello.com/c/uOaYnRzO

**Screenshots**

Travis CI running when a commit is pushed:

screen shot 2018-12-06 at 15 39 15

Added build status badge to repo README:

screen shot 2018-12-06 at 15 39 49

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  - postgresql
+
+before_install:
+  - "echo 'gem: --no-document' > ~/.gemrc"
+  # Solves rainbow issue: https://github.com/sickill/rainbow/issues/48
+  - gem update --system
+
+before_script:
+  - cp config/database.travis.yml config/database.yml
+  - psql -c "DROP DATABASE IF EXISTS \"wbooksapi_test\";" -U postgres
```
Comment (Gmarn) [inline .travis.yml:21]: Why did you remove the User creation step?

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  - psql -c "CREATE DATABASE \"wbooksapi_test\" ENCODING 'utf8';" -U postgres
+  - bundle exec rake db:migrate
+
+script:
+  # - bundle exec rspec spec -fd
```
Comment (Gmarn) [inline .travis.yml:26]: I would like travis pass with this line uncomment :)

File: .travis.yml [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+env:
+  global:
+    - RUBY_GC_MALLOC_LIMIT=90000000
+    - RUBY_GC_HEAP_FREE_SLOTS=200000
+
+services:
+  - postgresql
+
+before_install:
```
Comment (Gmarn) [inline .travis.yml:13]: You can keep this line because in few card we will add some specs.
- "echo '--colour' > ~/.rspec"

File: .travis.yml (hunk 1 above)
Comment (storresy) [reply .travis.yml:21]: I removed the User creation step because I'm using the default PG user to access the test DB. 

In the rails-bootstrap project, the travis database config file is also using the default PG user so I think the user creation step is not necessary in that project. Find bellow the links to the travis config and travis db config files in the rails-bootstrap project.

https://github.com/Wolox/rails-bootstrap/blob/master/.travis.yml
https://github.com/Wolox/rails-bootstrap/blob/master/config/database.travis.yml

File: .travis.yml (hunk 2 above)
Comment (Gmarn) [reply .travis.yml:26]: Ok, let it as it.

File: .travis.yml (hunk 1 above)
Comment (Gmarn) [reply .travis.yml:21]: Ok, we will try it ;)

**Review by Gmarn — APPROVED** [review]
